File: video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.js
// NOTE: This example uses the next generation Twilio helper library - for more
// information on how to download and install this version, visit
// https://www.twilio.com/docs/libraries/node
var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var apiKeySecret = 'your_api_key_secret';
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var Twilio = require('twilio');

var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});

client.video.rooms('DailyStandup').participants.get('Alice').publishedTracks.get('Camera').fetch()
  .then(publishedTrack => {
    console.log(publishedTrack.kind);
  });

File: video/rest/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.cs
using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Video.V1.Room.Participant;

class Program
{
    static void Main(string[] args)
    {
        // Find your API Key SID and Secret at twilio.com/console
        const string apiKeySid = "SKXXXX";
        const string apiKeySecret = "your_api_key_secret";

        PublishedTrackResource publishedTrack = PublishedTrackResource.Fetch(
                       "DailyStandup",
                       "Alice",
                       "Camera");

        Console.WriteLine(publishedTrack.Sid);
    }
}
